firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -33,17 +35,21 @@ public class FeedbackActivity extends AppCompatActivity {
 
   public static final String RELEASE_NAME_EXTRA_KEY =
       "com.google.firebase.appdistribution.FeedbackActivity.RELEASE_NAME";
+  public static final String INFO_TEXT_EXTRA_KEY =
+      "com.google.firebase.appdistribution.FeedbackActivity.INFO_TEXT";
   public static final String SCREENSHOT_FILENAME_EXTRA_KEY =
       "com.google.firebase.appdistribution.FeedbackActivity.SCREENSHOT_FILE_NAME";
 
   private FeedbackSender feedbackSender;
   private String releaseName;
+  private CharSequence infoText;
```

Comment:
Just curious, how do you choose `CharSequence` instead of a plain ole' `String`?

Reply:
Because that's what `Context.getText()` returns: https://developer.android.com/reference/android/content/Context#getText(int)

`Context.getString()` returns a String but it does not retain any html/styles, which we need.